### PR TITLE
client_lb_end2end_test: add OrcaLoadReport into EchoRequest proto

### PR DESCRIPTION
**Pull request by nicolasnoble**
I hate making giant PRs so here's one first small chunk for the load reporter work.

### CODE REVIEW

**Review by markdroth — APPROVED** [review]
Looks great!

File: src/proto/grpc/testing/echo_messages.proto [hunk 1]
```diff
@@ -51,6 +53,7 @@ message RequestParams {
   int32 backend_channel_idx = 16; // which backend to send request to
   bool echo_metadata_initially = 17;
   bool server_notify_client_when_started = 18;
+  xds.data.orca.v3.OrcaLoadReport send_back_report = 19;
```
[the remark this answers is not in the document]
Comment (nicolasnoble) [reply src/proto/grpc/testing/echo_messages.proto]: Done.

**Comment by nicolasnoble** [comment]
Not exactly sure why yet this isn't building correctly in some cases.

**Comment by dennycd** [comment]
for iOS test failure 

```
/Volumes/BuildData/tmpfs/altsrc/github/grpc/workspace_objc_macos_opt_native/test/cpp/ios/Pods/Headers/Private/RemoteTestCpp/src/proto/grpc/testing/echo_messages.pb.h:34:10: fatal error: 'src/proto/grpc/testing/xds/v3/orca_load_report.pb.h' file not found
#include "src/proto/grpc/testing/xds/v3/orca_load_report.pb.h"

```

looks like you need to also add the orca_load_report proto to 

https://github.com/grpc/grpc/blob/b92f8857569bd78ddd071e883db5edc94ecd594b/test/cpp/ios/RemoteTestClientCpp/RemoteTestCpp.podspec#L36

**Comment by nicolasnoble** [comment]
@dennycd: isn't it what I've done in https://github.com/grpc/grpc/pull/29505/commits/391f3608a0687fdf9447c6eef03e2f830ed6ae2a ?

**Comment by dennycd** [comment]
Ah, I see. It looks like we didn't pick up the generated proto header which is in a subdirectory. @nicolasnoble  can you try 

https://github.com/grpc/grpc/blob/2d4f3c56001cd1e1f85734b2f7c5ce5f2797c38a/test/cpp/ios/RemoteTestClientCpp/RemoteTestCpp.podspec#L17 

```
  s.source_files = "src/proto/grpc/testing/**/*.pb.{h,cc}"

```

I think the original podspec just assumed all generated source are directly under src/proto/grpc/testing. ;  )

**Comment by nicolasnoble** [comment]
Oooh, I see. Yep, trying this now. Thanks!

**Comment by nicolasnoble** [comment]
Alright, at this point all the tests are passing properly, and the copybara test is just getting confused about the internal import state. A manual cherry pick seems to confirm internal tests are okay too. Merging this one.

**Pull request merged**